syft/frameworks/torch/mpc/przs.py

```diff
@@ -5,7 +5,7 @@
 import torch
 import syft
 
-RING_SIZE = 2 ** 64
+RING_SIZE = 2 ** 5
```

Comment:
Q: should this be 5?

Reply:
it should be less than or equal the ring size of RST